data/assets/scene/solarsystem/planets/earth/satellites/misc/tiangong.asset

```diff
@@ -23,7 +23,7 @@ local omm = asset.resource({
 local TiangongPosition = {
   Identifier = "TiangongPosition",
   Parent = transforms.EarthInertial.Identifier,
-  BoundingSphere = 54.5, -- half the width
+  BoundingSphere = 27.8, -- half the width
```

Comment:
Why this change?

Reply:
Because this was straight copy pasted from the ISS asset and Tiangong is smaller. This is half the size I found on [Wikipedia](https://en.wikipedia.org/wiki/Tiangong_space_station). I felt I was not really able to fly close to it.

Reply:
Maybe this should be a separate commit?